rpcs3/Emu/Cell/lv2/sys_rsx.cpp

```diff
@@ -279,21 +279,52 @@ error_code sys_rsx_context_allocate(cpu_thread& cpu, vm::ptr<u32> context_id, vm
  * lv2 SysCall 671 (0x29F): sys_rsx_context_free
  * @param context_id (IN): RSX context generated by sys_rsx_context_allocate to free the context.
  */
-error_code sys_rsx_context_free(cpu_thread& cpu, u32 context_id)
+error_code sys_rsx_context_free(ppu_thread& ppu, u32 context_id)
 {
-	cpu.state += cpu_flag::wait;
+	ppu.state += cpu_flag::wait;
 
 	sys_rsx.todo("sys_rsx_context_free(context_id=0x%x)", context_id);
```

Comment:
Should it still be `todo`?

Reply:
I rather keep it as is for now.

Reply:
We can make it warn. Context release happens rather predictably